src/to-be-disabled.ts

```diff
@@ -57,7 +53,7 @@ export function toBeDisabled(element) {
   };
 }
 
-export function toBeEnabled(element) {
+export function toBeEnabled(this: jest.MatcherContext, element: ReactTestInstance) {
```

Comment:
I wonder why in case of some matchers we allow for `ReactTestInstance` and in others `ReactTestInstance | null`

Reply:
Similarly some of our matchers have explicit this typing while others do not have. If it's needed maybe apply it to all matchers. Wdyt?

Reply:
In some methods for example this we run
`reactTestInstance.toBeEnabled()` so we don't pass the element and we rely on RNTL typings.

In other methdos like `toContainElement` the consumer will do `reactTestInstance.toContainElement(anotherReactTestInstance)` and it make sense to make mandatory instead of allowing an undefined.